neqo-transport/src/cc/classic_cc.rs

```diff
@@ -430,67 +454,97 @@ mod tests {
+                1,                     // pn
+                time_before,           // time sent
+                true,                  // ack eliciting
+                Rc::default(),         // tokens
+                MAX_DATAGRAM_SIZE - 1, // size
+            ),
+            SentPacket::new(
+                PacketType::Short,
+                2,                     // pn
+                time_before,           // time sent
+                true,                  // ack eliciting
+                Rc::default(),         // tokens
+                MAX_DATAGRAM_SIZE - 2, // size
+            ),
+            SentPacket::new(
+                PacketType::Short,
+                3,                 // pn
+                time_before,       // time sent
+                true,              // ack eliciting
+                Rc::default(),     // tokens
+                MAX_DATAGRAM_SIZE, // size
+            ),
+            SentPacket::new(
+                PacketType::Short,
+                4,                 // pn
+                time_before,       // time sent
+                true,              // ack eliciting
+                Rc::default(),     // tokens
+                MAX_DATAGRAM_SIZE, // size
             ),
             SentPacket::new(
                 PacketType::Short,
-                2,             // pn
-                time_before,   // time sent
-                true,          // ack eliciting
-                Rc::default(), // tokens
-                105,           // size
+                5,                 // pn
+                time_before,       // time sent
+                true,              // ack eliciting
+                Rc::default(),     // tokens
+                MAX_DATAGRAM_SIZE, // size
             ),
             SentPacket::new(
                 PacketType::Short,
-                3,             // pn
-                time_after,    // time sent
-                true,          // ack eliciting
-                Rc::default(), // tokens
-                107,           // size
+                6,                 // pn
+                time_before,       // time sent
+                true,              // ack eliciting
+                Rc::default(),     // tokens
+                MAX_DATAGRAM_SIZE, // size
+            ),
+            SentPacket::new(
+                PacketType::Short,
+                7,                     // pn
+                time_after,            // time sent
+                true,                  // ack eliciting
+                Rc::default(),         // tokens
+                MAX_DATAGRAM_SIZE - 3, // size
             ),
         ];
 
-        cc.on_packet_sent(&sent_packets[0]);
-        assert_eq!(cc.acked_bytes, 0);
-        cwnd_is_default(&cc);
-        assert_eq!(cc.bytes_in_flight(), 103);
-
-        cc.on_packet_sent(&sent_packets[1]);
+        // Send some more packets so that the cc is not app-limited.
+        for p in sent_packets[..6].iter() {
```

Comment:
I think that clippy will remind you that you don't need to call `.iter()`.  Also, I think that you can use `&[]` rather than `vec![]` here.